.github/workflows/autoformat.yml

```diff
@@ -115,8 +115,8 @@ jobs:
       # ************************************************************************
       - if: steps.diff.outputs.python == 'true'
         run: |
-          ruff --fix .
-          black .
+          ruff check --fix .
+          ruff format .
           blacken-docs $(git ls-files '*.py' '*.rst' '*.md') || true
```

Comment:
We can remove blacken-docs once https://github.com/astral-sh/ruff/discussions/9871 is resolved.